src/main/java/org/terasology/dynamicCities/population/CultureComponent.java

```diff
@@ -58,4 +63,17 @@ public float getProcentualOfZone(String zone) {
         }
         return procentuals;
     }
+
+    public TownTheme getTownTheme() {
```

Comment:
Yeah, on top of that `Component` classes aren't actually meant to have code at all in the first place - they're meant to be pure data containers. Long time since I looked but I thought you could register cultures via prefabs?